src/components/Expander/examples/5.default-emphasis.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
You can get rid of these two stateless examples since we've stopped adding them for the most part.